src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoItem } from '../TodoItem/TodoItem';
+
+interface Props {
+  todos: Todo[];
+}
+
+export const TodoList: React.FC<Props> = React.memo(({ todos }) => {
+  return (
+    <section className="todoapp__main" data-cy="TodoList">
+      {todos.map(todo => <TodoItem todo={todo} />)}
```

Comment:
Each child in a list should have a unique 'key' prop. Add a unique key prop to the TodoItem component inside the map function.